src/Horarium.IntegrationTest/FallbackJobTest.cs

```diff
@@ -0,0 +1,73 @@
+using System.Threading.Tasks;
+using Horarium.IntegrationTest.Jobs;
+using Horarium.IntegrationTest.Jobs.Fallback;
+using Xunit;
+
+namespace Horarium.IntegrationTest
+{
+    [Collection(IntegrationTestCollection)]
+    public class FallbackJobTest : IntegrationTestBase
+    {
+        [Fact]
+        public async Task FallbackJobAdded_FallbackJobExecuted()
+        {
+            var horarium = CreateHorariumServer();
+
+            var mainJobRepeatCount = 2;
+            await horarium.Create<FallbackMainJob, int>(1)
+                          .MaxRepeatCount(mainJobRepeatCount)
+                          .AddRepeatStrategy<FallbackRepeatStrategy>()
+                          .AddFallbackConfiguration(configure =>
+                                                        configure.CreateFallbackJob<FallbackJob, int>(
+                                                            2,
+                                                            builder =>
+                                                            {
+                                                                builder.Next<FallbackNextJob, int>(3);
+                                                            }))
+                          .Schedule();
+            
+            await Task.Delay(7000);
+
+            horarium.Dispose();
+
+            var mainExecuted = FallbackMainJob.ExecutedCount;
+            var fallbackExecuted = FallbackJob.ExecutedCount;
+            var nextExecuted = FallbackNextJob.ExecutedCount;
+
+            FallbackMainJob.ExecutedCount = 0;
```

Comment:
Maybe do it in constructor? So every test will prepare data before run, and test code will be simplier (no need to store state in another variables and reset static variables then)